contrib/test/integration/vars.yml

```diff
@@ -0,0 +1,90 @@
+---
+
+# Enable using BATS to output integration-test xunit output (FIXME: broken? Kill it?)
+xunit: false
+
+rpm_pkgs:
+    - btrfs-progs-devel
+    - container-selinux
+    - curl
+    - device-mapper-devel
+    - expect
+    - findutils
+    - gcc
+    - git
+    - glib2-devel
+    - glibc-devel
+    - glibc-static
+    - golang
+    - gpgme-devel
+    - hostname
+    - iproute
+    - iptables
+    - krb5-workstation
+    - libassuan-devel
+    - libffi-devel
+    - libgpg-error-devel
+    - libguestfs-tools
+    - libseccomp-devel
+    - libvirt-client
+    - libvirt-python
+    - libxml2-devel
+    - libxslt-devel
+    - make
+    - mlocate
+    - nfs-utils
+    - nmap-ncat
+    - npm
+    - oci-register-machine
+    - oci-systemd-hook
+    - oci-umount
+    - openssl
+    - openssl-devel
+    - ostree-devel
+    - pkgconfig
+    - python
+    - python2-boto
+    - python2-crypto
+    - python2-mock
+    - python-click
+    - python-devel
+    - python-virtualenv
+    - PyYAML
+    - redhat-rpm-config
+    - rpcbind
+    - rsync
+    - sed
+    - skopeo-containers
+    - socat
+    - tar
+    - wget
+
+# For build/plugins.yml
+container_networking_plugins:
```

Comment:
This is a very static list I can't see someone needing to override, so moving it here potentially hurts the readability of the install scripts -- same for the `go get` items, etc.

Reply:
TBH I don't know how static they'll be.  For the moment though I would guess they're pretty stable.  I'm not married to having them here, other than it makes the include-file a little shorter to scroll-through.

Reply:
FWIW we've updated the list of Go tools we install for Origin AMIs twice in the last two years